README.md

```diff
@@ -1,7 +1,56 @@
 # BestFlow
 We are going to fix bad chatbots!
 
-Hosted at: https://bestflow.netlify.app/
+Diagnostic tool for chatbot transcripts that help retailers identify poor or strong questions and intents through an easily digestible and intuitive display.
 
-This project uses:
-React, Axios, Node, Tailwind, MongoDB, Express and more :D
+Created by Kaylee Chan, Emily Wan, Sam Weiss, Helia Sajjadian, and William Yao,
+along with the help of the University of Toronto's Technology Leadership Initiative professors and teaching assistants!
+
+## General Information:
+React, Axios, Node, Tailwind, MongoDB, Express, and more :D
+Licensed under the Apache-2.0 license.
+
+## Installation/How to Run:
+
+You can access our frontend by going to our hosted site: https://bestflow.netlify.app/
+
+Download and unzip our code in any folder of your choice, then open the unzipped folder in any JavaScript editor of your choice.
+In the project directory, you must open two terminals.
+
+### Backend Terminal
+
+In this terminal, run `cd back-end` to navigate to our backend folder, `npm install` to stay up-to-date,
+and then run `nodemon server` where our backend will start listening on port 5000.
+
```

Comment:
We should add that they need to create a .env file with
```
DATABASE_URI=[Private key, please contact one of us if you want to test this on your local build, otherwise use our hosted version]
TESTDATA_NS=Transcripts
API_KEY=[Private key that we do not currently use. Same as DATABASE_URI]
PORT=5000
```